lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -292,11 +337,15 @@ int main(int argc, char* argv[]) {
     proxy_service.wait_for_requests();  // block here instead of on server
     proxy_service.stop();               // stop queue after server shuts down
   });
-
+  MLOG(MINFO) << "start test server";
   // Block on main local_enforcer (to keep evb in this thread)
   local_enforcer->attachEventBase(evb);
+  MLOG(MINFO) << "Attached EventBase";
```

Comment:
shouldn't all those messages be `MDEBUG` level?

Reply:
converted

Reply:
Changed as MDEBUG